notifications_utils/logging.py

```diff
@@ -100,6 +101,9 @@ def get_handlers(app):
     if app.debug:
         handler = logging.StreamHandler(sys.stderr)
         handlers.append(configure_handler(handler, app, standard_formatter))
+    elif app.config['CLOUDFOUNDRY']:
```

Comment:
Usually a feature switch is better for a certain functionality, therefore I think it's better to use LOGGING_STDOUT_JSON=1, or LOGGING=stdout-json or similar.